user-guide/Advanced_Modules/Data_Sources/Tutorials/Data_API_GettingStarted.md

```diff
@@ -0,0 +1,61 @@
+﻿---
+uid: Data_Sources_Tutorials_GettingStarted
+---
+
+# Creating a scripted connector
+
+This tutorial will teach you how to create your first scripted connector with Python.
```

Comment:
Could you also explain what the purpose of this specific scripted connector is?

Reply:
Brief explanation added